remote/inc/patterns/subscribe.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Subscription form
+ */
+return array(
+	'title'      => __( 'Subscription form', 'remote' ),
+	'categories' => array( 'text', 'featured' ),
+	'content'    => '<!-- wp:paragraph {"style":{"typography":{"textTransform":"uppercase"}}} -->
```

Comment:
I think this should also have the Small font size preset, if I'm interpreting the design correctly.

Reply:
you are right!